Java/E-montazysta/src/main/java/com/emontazysta/model/dto/LocationDto.java

```diff
@@ -14,13 +16,21 @@
 public class LocationDto {
 
     private Long id;
+    @NotBlank(message = "Name cannot be empty")
     private String name;
+    @NotNull(message = "X coordinate cannot be empty")
     private Double xCoordinate;
+    @NotNull(message = "Y coordinate cannot be empty")
     private Double yCoordinate;
+    @NotBlank(message = "City cannot be empty")
     private String city;
+    @NotBlank(message = "Street cannot be empty")
     private String street;
+    @NotBlank(message = "Property number cannot be empty")
     private String propertyNumber;
+    @NotBlank(message = "Apartment number cannot be empty")
     private String apartmentNumber;
+    @NotBlank(message = "Zip code cannot be empty")
     private String zipCode;
```

Comment:
We should discuss it with analysis team.
We want to use x&y cords to open maps, so it's okay.
But there is possibility, that location will not have street (eg. in village), or apartment number.